src/catkin_pkg/python_setup.py

```diff
@@ -99,9 +102,16 @@ def generate_distutils_setup(package_xml_path=os.path.curdir, **kwargs):
         data['url'] = package.urls[0].url
 
     if len(package.description) <= 200:
-        data['description'] = package.description
+        if '\n' in package.description:
+            data['description'] = package.description.replace('\n', ' ')
+            data['long_description'] = package.description
+        else:
+            data['description'] = package.description
     else:
-        data['description'] = package.description[:197] + '...'
+        if '\n' in package.description:
+            data['description'] = package.description[:197].replace('\n', ' ') + '...'
+        else:
+            data['description'] = package.description[:197] + '...'
```

Comment:
```
    description = package.description.replace('\n', ' ')
    if len(description) <= 200:
        data['description'] = description
    else:
        data['description'] = description[:197] + '...'
```

Reply:
that's not the same if you want to set long_description only when it's different than description, but you're right that adding
```
if data['description'] != package.description
    data['long_description'] = package.description
```
after this section is probably more readable.

Reply:
maybe even do:
```
if len..
else..
data['description'] = data['description'].replace('\n', ' ')
```
I don't think we should set `long_description` if it only differs by newlines.